doc/examples/filters/plot_deconvolution.py

```diff
@@ -2,54 +2,114 @@
 =====================
 Image Deconvolution
 =====================
-In this example, we deconvolve an image using Richardson-Lucy
-deconvolution algorithm.
 
-The algorithm is based on a PSF (Point Spread Function),
-where PSF is described as the impulse response of the 
-optical system. The blurred image is sharpened through a number of 
-iterations, which needs to be hand-tuned.
+In this example, Wiener, unsupervised Wiener and Richardson-Lucy
+algorithms are used to deconvolve an image.
 
-.. [1] William Hadley Richardson, "Bayesian-Based Iterative 
-       Method of Image Restoration",
-       J. Opt. Soc. Am. A 27, 1593-1607 (1972), DOI:10.1364/JOSA.62.000055
+Wiener and unsupervised Wiener filters
+--------------------------------------
+First a noisy version of an image is deconvolved using Wiener
+and unsupervised Wiener algorithms. These algorithms are based on
+linear models that can't restore sharp edges as much as non-linear
+methods (like TV restoration), but are much faster.
 
-.. [2] http://en.wikipedia.org/wiki/Richardson%E2%80%93Lucy_deconvolution
+The Wiener filter is an inverse filter based on a PSF (Point Spread
+Function), the prior regularization (penalization of high frequency)
+and the tradeoff between the data and prior adequacy. The regularization
+parameter must be hand tuned.
+
+The unsupervised Wiener algorithm has self-tuned regularization
+parameters based on data learning. This is not common and based on the
```

Comment:
@stefanv I'd need to say, I don't know. I only joined the documentation, and I don't know what the original author meant. I would need to research a little before asking these questions and improving the documentation.